.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,38 @@
+name: GH Pages Deploy
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - README.md
+
+jobs:
+  build-deploy:
+    name: Build and deploy
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: main
+          path: source
+
+      - uses: actions/checkout@v2
```

Comment:
I think this one failed, probably because the gh-pages branch does not exist right now...
What did you find out @sergiomsantos and @Pbarbosa92 on the benchmarks repo?